API.md

```diff
@@ -355,43 +355,46 @@ public struct HTTPResponseStatus: Equatable, CustomStringConvertible, Expressibl
 }
 
 /// HTTP Methods handled by http_parser.[ch] supports
-public enum HTTPMethod : RawRepresentable {
-    /* be future-proof, http_parser can be upgraded */
-    case other(String)
+public struct HTTPMethod : RawRepresentable {
```

Comment:
I'll update to remove `RawRepresentable` and add `ExpressibleByStringLiteral` conformance.